src/frontend/src/flows/dappsExplorer/dapps.ts

```diff
@@ -12,8 +14,20 @@ type ElementOf<Arr> = Arr extends readonly (infer ElementOf)[]
 export type DappDescription = ElementOf<typeof dappsJson>;
 
 // The list of dapps we showcase
-export const dapps: DappDescription[] = dappsJson.map((dapp) => ({
-  ...dapp,
-  /* fix up logo path (inherited from dfinity/portal) to match our assets */
-  logo: dapp.logo.replace("/img/showcase/", "/icons/"),
-}));
+export const getDapps = async (): Promise<DappDescription[]> => {
+  const dapps: typeof dappsJson | null = (
```

Comment:
I actually have no idea, I decided to play it safe based on https://github.com/dfinity/gix-components/blob/14cf17fa63d59af95588b372dff4071abaf31568/src/lib/components/QRCode.svelte#L46-L63 (when I looked at the code it was `null` though now seems to be `undefined`)

Reply:
In gix-cmp I did not added any try/catch because the build fails if the path is incorrect but, that's a good point I can add a try/catch there as well for extra safety.